components/gantry/oneaxis/oneaxis.go

```diff
@@ -358,30 +358,28 @@ func (g *oneAxis) MoveToPosition(ctx context.Context, positions []float64, extra
 			return err
 		}
 
-		// Hits backwards limit switch, goes in forwards direction for two revolutions
-		if hit {
-			if x < g.positionLimits[0] {
-				dir := float64(1)
-				return g.motor.GoFor(ctx, dir*g.rpm, 2, extra)
+		// Hits backwards limit switch, and stops if position x is past the limit switch
+		if (hit == true) || (x <= (g.positionLimits[0] + 0.25)) {
```

Comment:
Any "magic" numbers should be constants at the top of the file with a name that describes what they are